File: T.I.2.4PAO202551_EUREKABANK_BDDNUBE_REST_DOTNET_GR01/04. SERVIDOR/SERV_EurekaBank_RESTDOTNET/ec.edu.pinza.modelo/Movimiento.cs
using System;

namespace ec.edu.pinza.modelo
{
    public class Movimiento
    {
        public string Cuenta { get; set; }
        public int NroMov { get; set; }
        public DateTime Fecha { get; set; }
        public string Tipo { get; set; }
        public string Accion { get; set; }
        public double Importe { get; set; }

        public Movimiento() { }

        public Movimiento(string cuenta, int nromov, DateTime fecha, string tipo, string accion, double importe)
        {
            Cuenta = cuenta;
            NroMov = nromov;
            Fecha = fecha;
            Tipo = tipo;
            Accion = accion;
            Importe = importe;
        }
    }
}

File: TI1.3_EUREKABANK_BDDNUBE_REST_DOTNET_GR01/04. CLIMOV/app/src/main/java/ec/edu/espe/rest_dotnet_bank/data/models/Movimiento.kt
package ec.edu.espe.rest_dotnet_bank.data.models

import com.google.gson.annotations.SerializedName

data class Movimiento(
    @SerializedName("NroMov")
    val nroMov: Int = 0,
    
    @SerializedName("Cuenta")
    val cuenta: String = "",
    
    @SerializedName("Fecha")
    val fecha: String = "",
    
    @SerializedName("Tipo")
    val tipo: String = "",
    
    @SerializedName("Accion")
    val accion: String = "",
    
    @SerializedName("Importe")
    val importe: Double = 0.0
)
